selenium_tests/homepage/test_search.py

```diff
@@ -0,0 +1,27 @@
+import time
+
+from selenium_tests.base import BaseSeleniumTests
+from home.factories import (
+    SectionIndexFactory,
+    SectionFactory,
+    ArticleFactory
+)
+
+from selenium_tests.pages import SearchPage
+
+class SearchSeleniumTests(BaseSeleniumTests):
+
+    def setUp(self):
+        self.setup_blank_site()
+        self.sections_index = SectionIndexFactory(parent=self.site.root_page)
+        self.section01 = SectionFactory(parent = self.sections_index)
+        self.article01 = ArticleFactory(parent=self.section01, title = 'Test article for search - asdf;lkj')
+
+    def test_search_bar(self):
+
+        home_page = self.visit_page(self.home)
```

Comment:
@sheralim012 was wondering if you could help me here. Have tried to set up a test just to do a basic search. But when I am running the test with selenium I get an internal server error. Clearly the search is working in the site so there is something wrong with my test setup, wondering if you had any ideas of what it could be.

![image](https://user-images.githubusercontent.com/69303397/206717928-143639d6-c191-4899-a792-5ca9261a53e3.png)

Reply:
To be fixed in a future PR.